### DISCUSSION TITLE
Получение новостей с чужой группы

### DISCUSSION THREAD

**Discussion by Gplay666** [MrZillaGold/VK2Discord#243] [Q&A] [answered]
Пока я настраивал плагин, я понял, что я вроде как (поправьте если не прав) не могу получить токен чужой группы. И тут появляется вопрос, есть ли способ вообще через этот скрипт брать новости из чужих групп?

**Comment by egorprnn** [comment, marked as answer]
Используйте токен от страницы обычного пользователя, это описано в [**инструкции**](https://github.com/MrZillaGold/VK2Discord/wiki/%D0%A2%D0%BE%D0%BA%D0%B5%D0%BD-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D1%8B).
